web/src/components/netflow-record/record-panel.tsx

```diff
@@ -225,6 +225,12 @@ export const RecordPanel: React.FC<RecordDrawerProps> = ({
     }
   }, [record.labels._RecordType, t]);
 
+  const getSortedJSON = React.useCallback(() => {
+    const allKeys = new Set<string>();
+    JSON.stringify(record, (key, value) => (allKeys.add(key), value));
```

Comment:
This `stringify` call seems useless? (return value is ignored)
Could it be simplified with just: 
```
return JSON.stringify(record, Object.keys(record).sort(), 2); 
```
?

Reply:
that's a trick to list all keys of `record`
https://stackoverflow.com/a/53593328

else you would need to implement an [iterator to get subkeys](https://stackoverflow.com/a/16168003) (record.labels & record.fields)

Reply:
This then: ?

```
  const flat = { ...record.fields, ...record.labels };
  return JSON.stringify(flat, Object.keys(flat).sort(), 2);
```

(sorry but the double-stringify looks really hacky :-) )

Reply:
done